listen_to_mqtt.sh

```diff
@@ -66,126 +71,157 @@ listen_to_mqtt() {
           ;;
 
         *)
-          log_error "Invalid configuration request; topic:$topic vin:$vin msg:$msg"
+          log_error "Invalid configuration command request; vin:$vin topic:$topic msg:$msg"
           ;;
         esac
         ;;
 
       command)
         case $msg in
-        wake)
-          log_notice "Waking Up"
-          send_command $vin "-domain vcsec $msg"
+        autosecure-modelx)
+          send_command $vin $msg "Close falcon-wing doors and lock vehicle"
           ;;
-        trunk-open)
-          log_notice "Opening Trunk"
-          send_command $vin $msg
+        charge-port-close)
+          send_command $vin $msg "Close charge port"
           ;;
-        trunk-close)
-          log_notice "Closing Trunk"
-          send_command $vin $msg
+        charge-port-open)
+          send_command $vin $msg "Open charge port"
+          ;;
+        charging-schedule-cancel)
+          send_command $vin $msg "Cancel scheduled charge start"
           ;;
         charging-start)
-          log_notice "Start Charging"
-          send_command $vin $msg
+          send_command $vin $msg "Start charging"
           ;;
         charging-stop)
-          log_notice "Stop Charging"
-          send_command $vin $msg
+          send_command $vin $msg "Stop charging"
           ;;
-        charge-port-open)
-          log_notice "Open Charge Port"
-          send_command $vin $msg
-          ;;
-        charge-port-close)
-          log_notice "Close Charge Port"
-          send_command $vin $msg
+        climate-off)
+          send_command $vin $msg "Turn off climate control"
           ;;
         climate-on)
-          log_notice "Start Climate"
-          send_command $vin $msg
+          send_command $vin $msg "Turn on climate control"
           ;;
-        climate-off)
-          log_notice "Stop Climate"
-          send_command $vin $msg
+        drive)
+          send_command $vin $msg "Remote start vehicle"
           ;;
         flash-lights)
-          log_notice "Flash Lights"
-          send_command $vin $msg
+          send_command $vin $msg "Flash lights"
           ;;
         frunk-open)
-          log_notice "Open Frunk"
-          send_command $vin $msg
+          send_command $vin $msg "Open vehicle frunk"
           ;;
         honk)
-          log_notice "Honk Horn"
-          send_command $vin $msg
+          send_command $vin $msg "Honk horn"
+          ;;
+        list-keys)
+          send_command $vin $msg "List public keys enrolled on vehicle"
           ;;
         lock)
-          log_notice "Lock Car"
-          send_command $vin $msg
+          send_command $vin $msg "Lock vehicle"
+          ;;
+        media-toggle-playback)
+          send_command $vin $msg "Toggle between play/pause"
+          ;;
+        ping)
+          send_command $vin $msg "Ping vehicle"
```

Comment:
I don't know what this one does in fact, and how we can use it.

Reply:
It can be 2 things, we'll need to test... I could be wrong but I think this is to know if the car is awake or not. We can use it to know if the car is present... but I think ping = awake.

Reply:
you have a new command that actually feeds back real data.
https://github.com/tesla-local-control/tesla_ble_mqtt_core/issues/25

Ping does output nothing at all... Maybe a successful command means the car is awake and key paired...

Reply:
@baylanger mark resolved when you have read it